app/managers.py

```diff
@@ -0,0 +1,43 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.table_name = "actors"
+        self._connection = sqlite3.connect("../cinema.sqlite")
+
+    def all(self) -> list[Actor]:
+        all_lines = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [Actor(*line) for line in all_lines]
+
+    def create(self, **kwvalues) -> None:
+        columns = ", ".join(kwvalues.keys())
+        placeholders = ", ".join([f":{key}" for key in kwvalues.keys()])
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} ({columns}) "
+            f"VALUES ({placeholders})",
+            kwvalues
+        )
+        self._connection.commit()
+
+    def update(self, id: int, new_first_name: str, new_last_name: str) -> None:
+        self._connection.execute(
+            f"UPDATE {self.table_name} "
```

Comment:
Use triple quotes for multiline queries + check sql formatting. This resource may help you https://sqlformat.org/